index.src.html

```diff
@@ -635,25 +641,50 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   <h2 id="integration-with-css">Integration with CSS</h2>
 
   The CSS Standard does not specify how it fetches resources referenced from
-  stylesheets. However, implementations use the URL a given stylesheet was
-  loaded from as referrer for the requests for resources from that stylesheet.
+  stylesheets. However, implementations should be sure to set the
+  referrer-related properties of any <a for="/">requests</a> initiated by stylesheets
+  as follows:
 
-  Implementations should keep track of a referrer policy for each stylesheet
-  that should be used to create requests for resources from the respective
-  stylesheet.
+  <ol>
+    <li>
+       If a <a>CSS declaration block</a> is responsible for the request,
```

Comment:
yeah, the rest of the spec also uses <ol> so I figured to stick with that for consistency